osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -53,7 +54,9 @@ def create_job(self, extraction_order):
         bounding_geometry = extraction_order.excerpt.bounding_geometry.subclass_instance
 
         request_data = OrderedDict({
-            "callback_url": "http://example.com",
+            "callback_url": "http://example.com{0}".format(
```

Comment:
we should ensure that this is in fact not _http://example.com_ in production. Would this change belong in this pull request as well?